sdk/cognitiveservices/Knowledge.QnAMaker/src/Microsoft.Azure.CognitiveServices.Knowledge.QnAMaker.csproj

```diff
@@ -3,8 +3,7 @@
     <AssemblyTitle>Microsoft Cognitive Services QnAMaker SDK</AssemblyTitle>
     <AssemblyDescription>Provides access to the Microsoft Cognitive Services QnAMaker APIs.</AssemblyDescription>
     <Description>This client library provides access to the Microsoft Cognitive Services QnAMaker APIs.</Description>
-    <Version>2.1.0-preview.2</Version>
-    <ApiCompatVersion>2.0.1</ApiCompatVersion>
```

Comment:
Please leave this in -- this ensures that we are not introducing backward compatible breaks.

Reply:
Discussed offline - Agreed to remove this in this version of SDK.